core/kernel/user_ta.c

```diff
@@ -128,6 +128,26 @@ static void dec_recursion(void)
 	tsd->syscall_recursion--;
 }
 
+static bool safe_to_call_utee_ta_entry(enum utee_entry_func func,
+				       uint32_t ta_flags)
+{
+	bool safe = false;
+
+	switch (func) {
+	case UTEE_ENTRY_FUNC_OPEN_SESSION:
+	case UTEE_ENTRY_FUNC_CLOSE_SESSION:
+	case UTEE_ENTRY_FUNC_INVOKE_COMMAND:
+		safe = true;
```

Comment:
`return true;`

and `return false;` on `default:`

Reply:
sorry, didn't get your point on this. can you put more detail of this comment?